### PR TITLE
Add newtab interactions entry

### PR DESCRIPTION
**Pull request by mmccorks**

### CODE REVIEW

**Comment by badboy** [comment]
You will have to add it to `SUMMARY.md` to show up somewhere.

File: src/datasets/other/newtab/reference.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Scheduling:  
+"newtab" pings can be sent for one of two reasons. 
+component_init: The newtab component init'd, and the newtab and homepage settings have been categorized. This is mostly to ensure we hear at least once from clients configured to not show a newtab UI.
+newtab_session_end: The newtab visit ended. Could be by navigation, being closed, etc. 
+
+This dataset does not have a long history. The newtab ping went live in Fall 2022 and therefore we do not have data prior to that launch.  Any historical analysis will not be possible until we collect more data. 
+
+Currently, telemetry is not yet instrumented for organic topsites tiles. This work is in progress and will be added to the dataset soon.  For now, all total topsites metrics (impression, clicks) only include sponsored topsites data. 
+
+Some of the preference settings enabling features like Pocket and Topsites can be set to `enabled` in regions where these features are not available. In these cases, it is best to filter by country to only include regions where these features are available. In Looker, there are pre-set country groups for Topsties Available and Pocket Available to make this easier.
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:22]:
```suggestion
Some of the preference settings enabling features like Pocket and Topsites can be set to `enabled` in regions where these features are not available. In these cases, it is best to filter by country to only include regions where these features are available. In Looker, there are pre-set country groups for Topsites Available and Pocket Available to make this easier. 
```

File: src/datasets/other/newtab/reference.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+This dataset does not have a long history. The newtab ping went live in Fall 2022 and therefore we do not have data prior to that launch.  Any historical analysis will not be possible until we collect more data. 
+
+Currently, telemetry is not yet instrumented for organic topsites tiles. This work is in progress and will be added to the dataset soon.  For now, all total topsites metrics (impression, clicks) only include sponsored topsites data. 
+
+Some of the preference settings enabling features like Pocket and Topsites can be set to `enabled` in regions where these features are not available. In these cases, it is best to filter by country to only include regions where these features are available. In Looker, there are pre-set country groups for Topsties Available and Pocket Available to make this easier. 
+
+## Data Reference 
+
+### Field Descriptions
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:26]: We should add these directly to the schema as well. Pages like this tend to become outdated somewhat quickly

File: src/datasets/other/newtab/reference.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+`pocket_impressions`: count of total impressions on Pocket tiles, including both organic and sponsored. Each tile displayed contributes to this count.
+`sponsored_pocket_impressions`: the count of sponsored Pocket impressions
+`organic_pocket_impressions`: the count of organic Pocket impressions
+`pocket_clicks`: count of total Pocket clicks, including both organic and sponsored. 
+`sponsored_pocket_clicks`: the count of sponsored Pocket clicks
+`organic_pocket_clicks`: the count of organic pocket clicks
+`pocket_saves`: count of times “save to pocket” was selected on all tiles
+`sponsored_pocket_saves`: count of times “save to pocket” was selected on sponsored tiles
+`organic_pocket_saves`: count of times “save to pocket” was selected on organic tiles
+`pocket_topic_click`: count of clicks on “Popular Topics”
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:66]: This was a future field iirc?

File: src/datasets/other/newtab/reference.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+#### Topsites Specific:
+`topsites_impressions`: count of impressions on topsites tiles. Currently this is only instrumented for sponsored topsites and does not include non-sponsored topsites. 
+`sponsored_topsite_impressions`: count of impressions on sponsored topsites tiles. Each tile displayed contributes to this count. 
+`topsites_clicks`: count of clicks on topsites tiles. Currently this is only instrumented for sponsored topsites and does not include non-sponsored topsites. 
+`sponsored_topsite_clicks`: count of clicks on sponsored topsites tiles
+
+
+## Scheduling
+This dataset is updated daily via the telemetry-airflow infrastructure.
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:77]:
```suggestion
This dataset is [updated daily](https://workflow.telemetry.mozilla.org/dags/bqetl_newtab) via the telemetry-airflow infrastructure. 
```

File: src/datasets/other/newtab/reference.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+
+## Scheduling
+This dataset is updated daily via the telemetry-airflow infrastructure. 
+
+## Schema
+The data is partitioned by `submission_date`. 
+
+## Code Reference 
+This dataset is generated by bigquery-etl. Refer to this repository for information on how to run or augment this dataset.
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:83]:
```suggestion
This dataset is generated by bigquery-etl. Refer to [this repository](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql) for information on how to run or augment this dataset. 
```

File: src/datasets/other/newtab/reference.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
+# Newtab Interactions
```
Comment (ANich) [inline src/datasets/other/newtab/reference.md:1]: There are a few missing additions to the `.spelling` file: https://github.com/mozilla/data-docs/actions/runs/4086382290/jobs/7045605705

Comment (mmccorks) [reply src/datasets/other/newtab/reference.md:1]: Thanks, I will update this.

**Review by chutten — APPROVED** [review]

**Pull request merged**